rosdep/python.yaml

```diff
@@ -434,6 +434,19 @@ python-adafruit-bno055-pip:
   ubuntu:
     pip:
       packages: [adafruit_bno055]
+python-albumentations-pip:
```

Comment:
```suggestion
python3-albumentations-pip:
```

Reply:
is this due to `kinetic` no longer being supported? so any `rosdep` key in the `python.yaml` is now being considered `python3`?

Reply:
> so any `rosdep` key in the `python.yaml` is now being considered `python3`?

It's a compromise amongst a number of bad options.  Essentially we always prefer the distribution/system key, and those are always named `python3-<something>`.  In order to make the pip key live "near" where the distribution key would live, we are naming all pip keys `python3-<something>-pip`.  This also represents the fact that python2 is no longer really supported, and will completely go out of support once the current host of LTS distributions goes EOL (probably ~2023, when Bionic goes EOL).